guides/dimensiondata.md

```diff
@@ -0,0 +1,721 @@
+---
+layout: page
+title: "Dimension Data: Getting Started Guide"
+permalink: /guides/dimensiondata/
+---
+
+[jclouds](http://jclouds.apache.org/) is an open source multi-cloud toolkit for the Java platform that gives you the freedom to create applications that are portable across clouds while giving you full control to use cloud-specific features.
+
+This guide will show you how to programmatically use the Dimension Data CloudControl provider in jclouds to perform common operations available in the CloudControl API.
+
+## Table of Contents
+
+* [Concepts](#concepts)
+* [Getting Started](#getting-started)
+* [Portable Abstraction Usage](#portable-abstraction-usage)
+* [How to: List a Data Center](#how-to-list-a-data-center)
+* [How to: Deploy a Network Domain](#how-to-deploy-a-network-domain)
+* [How to: Deploy a Vlan](#how-to-deploy-a-vlan)
+* [How to: List Os Images](#how-to-list-available-os-images)
+* [How to: Deploy a Server](#how-to-deploy-a-server)
+* [How to: Apply a Tag to an Asset](#how-to-apply-a-tag-to-an-asset)
+* [Examples](#examples)
+* [Support and Feedback](#support-and-feedback)
+
+--------------
+
+## <a id="concepts"></a>Concepts
+
+The jclouds library wraps the [Dimension Data CloudControl API](https://docs.mcp-services.net/display/DEV/Welcome+to+the+CloudControl+documentation+portal). All operations are performed over SSL and authenticated using your Dimension Data CloudControl credentials. The API can be accessed directly over the Internet from any application that can send an HTTPS request and receive an HTTPS response.
+
+
+## <a id="getting-started"></a>Getting Started
+
+Before you begin you will need to have a Dimension Data CloudControl account for your organization. The credentials will be used to authenticate against the API.
+
+The Dimension Data CloudControl provider supports [API version 2.4](https://docs.mcp-services.net/display/CCD/Understanding+API+v0.9+vs.+API+v2) and [MCP 2.0 datacenters](https://docs.mcp-services.net/display/CCD/Introduction+to+MCP+2.0+Data+Center+Locations) only. 
+
+Upgrading to later versions is planned and in progress, see [JCLOUDS-1456](https://issues.apache.org/jira/browse/JCLOUDS-1456) and [JCLOUDS-1407](https://issues.apache.org/jira/browse/JCLOUDS-1407) for current progress.
+
```

Comment:
Instead of linking to specific issues (which will become obsolete quite soon) you could add a generic message and point to a JIRA search on the `dimensiondata` label.

Reply:
Great idea, I will make this adjustment.